src/test/java/org/fundacionjala/trello/stepdefs/ListStepDef.java

```diff
@@ -49,9 +49,25 @@ public void iCreateAListWith(final Map<String, String> data) {
      *
      * @param expectedData expected data to validate the creation.
      */
-    @Then("I should have a list created with:")
-    public void iShouldHaveAListCreatedWith(final Map<String, String> expectedData) {
+    @Then("I should have a list (created)(updated) with:")
```

Comment:
consider changing the RegEx to `(created|updated)`

Reply:
I tried to use that RegEx but it does not work